### ISSUE TITLE
Workflow RegEx Compilation failed

### ISSUE THREAD

**Issue by githubuser-112** [freescout-helpdesk/freescout#3051]
Hi,

we´re having errors showing in the log for RegEx we are using in the workflows. The workflows are working fine though, so patterns we want to check for are correctly identified.

Here´s the error: 
```
Invalid Workflow conditions regex: =?UTF-8?q?/.*(storno|stornieren|wie?derruf(en?)?).*/i?=. preg_match(): Compilation failed: quantifier does not follow a repeatable item at offset 0
```

The "?UTF-8?q?" part is not from us, in the Workflow we only put "/.\*(storno|stornieren|wie?derruf(en?)?).\*/i". 

The error must be for the first "?" already, as the quantifier has no item. But why is this even there and how can we stop this from happening?

Thank you very much!

PHP version: 8.2.6
FreeScout version: 1.8.77

**Comment by freescout-helpdesk** [comment]
Please attach screenshot of the Workflow condition.

**Comment by githubuser-112** [comment]
Thanks, here the screenshot:

![Workflow Condition](https://github.com/freescout-helpdesk/freescout/assets/134635748/05fb6796-feab-4a48-a52b-0a30a16e6e19)

**Comment by freescout-helpdesk** [comment]
Fixed in Workflows Module v1.0.44
